porch/controllers/packagevariants/config/rbac/role.yaml

```diff
@@ -31,18 +31,6 @@ rules:
   - patch
   - update
   - watch
-- apiGroups:
```

Comment:
I think this is probably needed. Perhaps there's a missing gen directive?

Reply:
Yeah, I saw those. I was going to check with you or @natasha41575 before merging. I can update the directives to include this.